script/lib/run-apm-install.js

```diff
@@ -8,9 +8,7 @@ module.exports = function(packagePath, ci, stdioOptions) {
   const installEnv = Object.assign({}, process.env);
   // Set resource path so that apm can load metadata related to Atom.
   installEnv.ATOM_RESOURCE_PATH = CONFIG.repositoryRootPath;
-  // Set our target (Electron) version so that node-pre-gyp can download the
-  // proper binaries.
-  installEnv.npm_config_target = CONFIG.appMetadata.electronVersion;
+
```

Comment:
👍 

apm should indeed be setting this very same env var here:

https://github.com/atom/apm/blob/1384050001726904ee00b30b596df57e0169e131/src/command.coffee#L110

Reply:
Oops, I see you already linked this same line of source code. My bad.